pkg/controller/service_test.go

```diff
@@ -20,33 +19,33 @@ import (
 )
 
 type portMapperMock struct {
-	findFunc   func(svc k8s.ServicePort) (int32, bool)
-	addFunc    func(svc *k8s.ServicePort) (int32, error)
-	removeFunc func(svc k8s.ServicePort) (int32, error)
+	findFunc   func(ns, name string, port int32) (int32, bool)
```

Comment:
Can you rename all ns param in this file?

```suggestion
	findFunc   func(namespace, name string, port int32) (int32, bool)
```